backend/core/src/seeder/seed.ts

```diff
@@ -221,15 +221,6 @@ const seedListings = async (
       (jurisdiction) => jurisdiction.name === listing.jurisdictionName
     )
     listing.leasingAgents = [everyOtherAgent]
-    const applicationMethods = await applicationMethodsService.create({
```

Comment:
This method is not valid data

Reply:
What does this mean?

Reply:
If we don't need this, there's an unused import and `applicationMethodsService` function that can be removed too.